### ISSUE TITLE
Additional CSS not visible in editor on List block

### ISSUE THREAD

**Issue by ijerkov** [WordPress/gutenberg#46134]
### Description

List block doesn't have Additional CSS classes loaded in the Gutenberg editor.
For test, I opened a post that has Additional classes on List, they are not visible in the editor.
Then I just saved the post without making any change.The result:

Before save:
```
<!-- wp:list {"className":"warnings-red spaced-half"} -->	
<ul class="warnings-red spaced-half">
```
After save:
```
<!-- wp:list -->
<ul><!-- wp:list-item -->
```
This means that after every old post update, we have to add back additional classes.

### Step-by-step reproduction instructions

Open existing post that have a list with additional css class in new wp version 6.1.1.
If you write the class name and save, it will work. className is lost somewhere in the process of rewriting wp:list to add  wp:list-item 

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by AntoninoBonanno** [comment]
Same problem

**Comment by t-hamano** [comment]
Thanks for the report.
Similar problems have been reported with #45920 and #45987 and have been resolved by #46000 (included in Gutenberg 14.7).

I would like to close this issue, but if the issue has not been resolved, please do not mind commenting.
